website/content/docs/concepts/vault-server-temporarily-overloaded.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+page_title: Vault server temporarily overloaded
+description: |-
+  Vault Enterprise error when the request limiter is at capacity.
+---
+
+# Vault server temporarily overloaded
+
+Vault returns a `503 - Service Unavailable` response to indicate that a request
+was rejected after Vault has reached its in-flight request capacity:
+
+```
+Error making API request.
+
+URL: PUT https://127.0.0.1:61555/v1/auth/userpass/login/foo
+Code: 503. Errors:
+
+* 1 error occurred:
+	* Vault server temporarily overloaded
+```
+
+`503 - Service Unavailable` is a retryable HTTP error, which is handled
+internally by the Vault API.
```

Comment:
This makes it sound like the Vault server will handle retrying. Can we instead clarify that the client in the api package handles retries?

Reply:
Good call! Addressed in https://github.com/hashicorp/vault/pull/25557/commits/7a38c1ff7ee0448f14b115478f967947a06e704e